clash-cores/test/Test/Cores/I2C/Config.hs

```diff
@@ -49,7 +52,7 @@ configT
   -> SimIO ConfO
 configT s0 ConfI{i2cRst=rst,i2cEna=ena,i2cCmdAck=cmdAck,i2cRxAck=rxAck,i2cAl=al} = do
   s <- readReg s0
-  let ConfS confStateM claim op lutIndex fault = s
+  let ConfS confStateM claim op lutIndex fault debug = s
```

Comment:
Why are debug messages made conditional? Perhaps that could be mentioned in the commit message.

Reply:
Because otherwise they flood the terminal when you run `clash-cores:unittests` :)